editor/editor_run.cpp

```diff
@@ -227,6 +227,12 @@ Error EditorRun::run(const String &p_scene, const String &p_write_movie) {
 	String shortcut;
 	VariantWriter::write_to_string(ED_GET_SHORTCUT("editor/stop_running_project"), shortcut);
 	OS::get_singleton()->set_environment("__GODOT_EDITOR_STOP_SHORTCUT__", shortcut);
+	// Pass the debugger run default scene shortcut to the running instance(s).		//changed by andre
```

Comment:
```suggestion
	// Pass the debugger run default scene shortcut to the running instance(s).
```

Reply:
Do I have to make the changes or do you make them?

Reply:
These are just suggestions, you need to do them yourself

Reply:
Surely, do i need to open a new PR ?

Reply:
No just update this branch